docs/source/guide/distributed-preprocessing.rst

```diff
@@ -4,9 +4,18 @@
 ------------------------------------------
 
 Before launching training jobs, DGL requires the input data to be partitioned
-and distributed to the target machines. For relatively small graphs, DGL
-provides a partitioning API :func:`~dgl.distributed.partition_graph` that
-partitions an in-memory :class:`~dgl.DGLGraph` object. It supports
+and distributed to the target machines. In order to handle diffierent scales
+of graphs, DGL provides 2 partitioning approaches:
+
+* A partitioning API for relatively small graphs.
+* A distributed partition pipeline for relatively large graphs.
```

Comment:
`* A distributed partition pipeline for graphs beyond a single machine capacity.`